object.go

```diff
@@ -44,6 +44,10 @@ type ObjectStoreManager interface {
 	CreateObjectStore(cfg *ObjectStoreConfig) (ObjectStore, error)
 	// DeleteObjectStore will delete the underlying stream for the named object.
 	DeleteObjectStore(bucket string) error
+	// BucketNames is used to retrieve a list of bucket names
+	BucketNames(opts ...ObjectOpt) <-chan string
```

Comment:
Maybe StoreNames and Stores?

Reply:
Hm, makes sense as we mostly use `ObjectStore` instead of `Bucket` elsewhere. What about `ObjectStoreNames` and `ObjectStores`? Just to make it more obvious when using `JetStreamContext` interface.

Reply:
We need to keep StreamsInfo but let's make alias for Streams() and that will be consistent imo.